core/src/main/scala/org/apache/spark/deploy/mesos/MesosClusterDispatcherArguments.scala

```diff
@@ -35,6 +36,11 @@ private[mesos] class MesosClusterDispatcherArguments(args: Array[String], conf:
   propertiesFile = Utils.loadDefaultSparkProperties(conf, propertiesFile)
 
   private def parse(args: List[String]): Unit = args match {
+    case ("--webui-url" | "-u") :: value :: tail =>
```

Comment:
case ("--port" | "-p") :: IntParam(value) :: tail =>
      port = value
      parse(tail)

    case ("--webui-port" | "-p") :: IntParam(value) :: tail =>
      webUiPort = value
      parse(tail)
Same short  options for both: -p. This is not correct these are different ports. I propose not to have a short option like the --name long option which does not have a short one.